New doc for issues and workarounds

rajkaramchedu opened the pull request:
Let's populate and grow this doc with common issues and workarounds. I consolidated troubleshooting sections from a few docs into this top-level doc. The goal is to monitor the community channels and record the issues and workarounds on this page. Use this page as a first-response go-to link for community questions.

<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/aptos-labs/aptos-core/4642)
<!-- Reviewable:end -->

clay-aptos commented on developer-docs-site/docs/issues-and-workarounds.md:
```diff
@@ -0,0 +1,116 @@
+
+#### Workaround
+
+This likely means that the state of the install is out of sync with the saved terraform state file located in the storage bucket (configured during `terraform init` statement).  This could happen if the cluster or other components were deleted outside of terraform, or if terraform had an error and did not finish.  Use the following commands to check the state.  Delete the state that is related to ...
```
How will they know which state is related to the error message so they may delete it?

rajkaramchedu replied:
See below comment.

clay-aptos replied:
Thanks, resolving.

clay-aptos commented on developer-docs-site/docs/issues-and-workarounds.md:
```diff
@@ -0,0 +1,116 @@
+
+#### Description
+
+When starting a node with the `cargo run -p ...` command, you receive a "Unable to fetch any peers to poll" error. It looks like you have no peers on the available node testers. You have no output when running:
```
I thought I heard Greg say we no longer recommend `cargo` use.

rajkaramchedu replied:
I didn't spend any time editing it, just grabbed the troubleshooting sections scattered in multiple docs and put them here. Let's review it after we merge it. I am feeling a lack of time this week and prioritizing stuff fast.

clay-aptos replied:
OK, let's leave for now.

clay-aptos reviewed (approved):
Thanks, Raj. I made some small edits in a direct commit and made a couple of comments here. I do note the links appear broken here. Not sure about when they are in aptos.dev.

rajkaramchedu commented:
> Details

I thought I fixed all the links? Which ones are broken on this page?

clay-aptos commented:
> > Details
> 
> I thought I fixed all the links? Which ones are broken on this page?

Sorry, the [Add upstream seed peers](https://github.com/aptos-labs/aptos-core/blob/988977bbf1fa33b263f0bfbef794187d091b2d1f/nodes/full-node/fullnode-source-code-or-docker#add-upstream-seed-peers) links appear broken in GitHub.

rajkaramchedu commented:
> > > Details
> > 
> > 
> > I thought I fixed all the links? Which ones are broken on this page?
> 
> Sorry, the [Add upstream seed peers](https://github.com/aptos-labs/aptos-core/blob/988977bbf1fa33b263f0bfbef794187d091b2d1f/nodes/full-node/fullnode-source-code-or-docker#add-upstream-seed-peers) links appear broken in GitHub.

Ah, right. I wanted to talk to you about this. For some reason, when I used `/docs/...` for links they did not work on my local, it gave me page not found. So I reverted back to using `/nodes...` way of linking. Actually I rarely use GitHub link sanity but we should, you are right. Maybe we should take this issue separately, worth experimenting it with Docusaurus and settling on a solution.

rajkaramchedu commented:
Here is the dec preview. Links work okay here. https://deploy-preview-4642--aptos-developer-docs.netlify.app/issues-and-workarounds

clay-aptos commented:
> > > > Details
> > > 
> > > 
> > > I thought I fixed all the links? Which ones are broken on this page?
> > 
> > 
> > Sorry, the [Add upstream seed peers](https://github.com/aptos-labs/aptos-core/blob/988977bbf1fa33b263f0bfbef794187d091b2d1f/nodes/full-node/fullnode-source-code-or-docker#add-upstream-seed-peers) links appear broken in GitHub.
> 
> Ah, right. I wanted to talk to you about this. For some reason, when I used `/docs/...` for links they did not work on my local, it gave me page not found. So I reverted back to using `/nodes...` way of linking. Actually I rarely use GitHub link sanity but we should, you are right. Maybe we should take this issue separately, worth experimenting it with Docusaurus and settling on a solution.

I was able to fix by making the links relative and adding the Markdown extension. Still, I am not sure those changes will work in the live site. We should test. Please take a look. Thanks!

rajkaramchedu commented:
> > > > > Details
> > > > 
> > > > 
> > > > I thought I fixed all the links? Which ones are broken on this page?
> > > 
> > > 
> > > Sorry, the [Add upstream seed peers](https://github.com/aptos-labs/aptos-core/blob/988977bbf1fa33b263f0bfbef794187d091b2d1f/nodes/full-node/fullnode-source-code-or-docker#add-upstream-seed-peers) links appear broken in GitHub.
> > 
> > 
> > Ah, right. I wanted to talk to you about this. For some reason, when I used `/docs/...` for links they did not work on my local, it gave me page not found. So I reverted back to using `/nodes...` way of linking. Actually I rarely use GitHub link sanity but we should, you are right. Maybe we should take this issue separately, worth experimenting it with Docusaurus and settling on a solution.
> 
> I was able to fix by making the links relative and adding the Markdown extension. Still, I am not sure those changes will work in the live site. We should test. Please take a look. Thanks!

Perfect. I just rebased and it triggered a new build. Links work in the preview docs. Awesome, thanks!
